scripts/document.sh

```diff
@@ -32,12 +30,32 @@ jazzy \
     --podspec MapboxDirections.swift.podspec \
     --config docs/jazzy.yml \
     --sdk iphonesimulator \
-    --module-version ${SHORT_VERSION} \
-    --github-file-prefix "https://github.com/mapbox/MapboxDirections.swift/tree/${BRANCH}" \
+    --module-version ${VERSION} \
+    --github-file-prefix "https://github.com/mapbox/MapboxDirections.swift/tree/v${VERSION}" \
     --documentation=docs/guides/*.md \
-    --root-url "https://mapbox.github.io/mapbox-navigation-ios/directions/${RELEASE_VERSION}/" \
+    --root-url "https://mapbox.github.io/mapbox-navigation-ios/directions/${VERSION}/" \
     --theme ${THEME} \
-    --output ${OUTPUT}
+    --output "${OUTPUT}/${VERSION}"
 
-find ${OUTPUT} -name *.html -exec \
+find "${OUTPUT}/${VERSION}" -name *.html -exec \
     perl -pi -e 's/BRANDLESS_DOCSET_TITLE/Directions.swift $1/, s/MapboxDirections.swift\s+(Docs|Reference)/MapboxDirections.swift $1/' {} \;
+
+function parseSemver() {
+    local RE='[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)\([0-9A-Za-z-]*\)'
+    eval $2=`echo $1 | sed -e "s#$RE#\1#"` # major
+    eval $3=`echo $1 | sed -e "s#$RE#\2#"` # minor
+    eval $4=`echo $1 | sed -e "s#$RE#\3#"` # patch
+    eval $5=`echo $1 | sed -e "s#$RE#\4#"` # prerelease
```

Comment:
This is intense. 😀 I guess I would’ve tried to use a Bash array for this, but as long as the string being evaluated is coming purely from the script, then this should be safe.